lib/utils/promptForProject.js

```diff
@@ -0,0 +1,37 @@
+const { AutoComplete } = require('enquirer');
+
+const output = require('../output');
+
+function formatProjectChoice(project) {
+  return project.name + " " + output.subtle(
+    project.url || `https://beta.dittowords.com/doc/${project.id}`
+  ) 
+}
+
+function parseResponse(response) {
+  const [, name, id] = response.split(/^(.*)\s.*http.*\/(\w+).*$/);
+
+  if (id === 'all') {
+    return { name, id: 'ditto_component_library'}
+  }
```

Comment:
This code is from https://github.com/dittowords/cli/pull/13, and I moved it here during the process of resolving a merge conflict. It would be good to do some careful checking to ensure I didn't introduce a regression!

Reply:
I think this looks good -- thanks so much for calling this out! @XavierAgostini if you want to look at this also and see if it makes sense? 

I'll also do some testing as well to make sure thinks are working ok